pkgs/tools/misc/lbdb/default.nix

```diff
@@ -1,17 +1,39 @@
-{ stdenv, fetchurl, perl, finger_bsd }:
+{ stdenv, fetchurl, perl, perlPackages, finger_bsd, makeWrapper
+, abook ? null
+, gnupg ? null
+, goobook ? null
+, khard ? null
```

Comment:
Does it make sense to enable anything from this list by default?

Reply:
IMHO it doesn't make sense. At least on of them (AFAIK khard) will throw an error if no user configuration is available. But I am open to suggestions.